bokeh/models/tickers.py

```diff
@@ -66,7 +66,7 @@ class AdaptiveTicker(Ticker):
     The smallest allowable interval between two adjacent ticks.
     """)
 
-    max_interval = Float(100.0, help="""
+    max_interval = Float(float('Inf'), help="""
```

Comment:
Well `Infinity` is the correct default, at least conceptually. It's the one that was getting used, because it was the JS default, and we never sent the python value (since it was usually never changed) We could choose to interpret `None` / `null` as  "no max value", the same practically speaking as `Infinity` as a max value.